CONTRIBUTING.md

```diff
@@ -31,3 +31,41 @@ We use Jenkins to run several tests on each pull request.
 
 If you don't want to run a build for a particular pull request, because all you are changing is the
 README for example, add the text `[skip ci]` to the PR title.
+
+Good to know
+------------
+
+#### Sample files
+If you are modifying one of these files:
+
+- `./roles/ceph-mds/defaults/main.yml`
+- `./roles/ceph-mgr/defaults/main.yml`
+- `./roles/ceph-fetch-keys/defaults/main.yml`
+- `./roles/ceph-rbd-mirror/defaults/main.yml`
+- `./roles/ceph-agent/defaults/main.yml`
+- `./roles/ceph-defaults/defaults/main.yml`
+- `./roles/ceph-osd/defaults/main.yml`
+- `./roles/ceph-nfs/defaults/main.yml`
+- `./roles/ceph-client/defaults/main.yml`
+- `./roles/ceph-common/defaults/main.yml`
+- `./roles/ceph-iscsi-gw/defaults/main.yml`
+- `./roles/ceph-mon/defaults/main.yml`
+- `./roles/ceph-rgw/defaults/main.yml`
+- `./roles/ceph-restapi/defaults/main.yml`
+- `./roles/ceph-docker-common/defaults/main.yml`
+- `./roles/ceph-common-coreos/defaults/main.yml`
+
+You must run `./generate_group_vars_sample.sh` to generate the `group_vars/*.sample` files.
+
+
+#### Keep your branch up-to-date
```

Comment:
You can add that rebased branches are more likely to be integrated easily & shorter.